pkg/urbit/jets/a/mod.c

```diff
@@ -10,15 +10,10 @@
   u3qa_mod(u3_atom a,
            u3_atom b)
   {
-#if 0
-    if ( b == 3 && a == 2684227708 ) {
-      printf("dword at 0x27ff84ff8 is %" PRIu64 "\r\n", *(c3_d *)0x27ff84ff8);
-      *(c3_d *)0x27ff84ff8 = 25;
-      printf("see, we modified it\r\n");
-    }
-#endif
     if ( 0 == b ) {
       return u3m_bail(c3__exit);
+    } else if ( _(u3a_is_cat(a)) && _(u3a_is_cat(b)) ) {
+      return a % b;
```

Comment:
Nice catch! I bet there's quite a bit of this sort of low-hanging fruit lying around. I'd like to really audit these jets at some point ...